src/job/MomoJobDescription.ts

```diff
@@ -7,20 +8,25 @@ import { JobEntity } from '../repository/JobEntity';
  * running: the number of currently running executions
  */
 export interface JobSchedulerStatus {
-  interval: string;
+  schedule?: IntervalSchedule | CronSchedule;
```

Comment:
why is this optional now?

Reply:
Not changed by you, but now I am wondering ... why do we have the schedule in the MomoJobDescription AND in the JobSchedulerStatus (which is part of the MomoJobDescription) again?

Reply:
Maybe the schedule in the MomoJobDescription is the one that's written in the database and the one in JobSchedulerStatus the one the job is actually currently running on - which must not necessarily be the same if job was changed after start?

Reply:
Yeah, it doesn't need to be optional anymore